integration_tests/test_base_url.py

```diff
@@ -2,7 +2,7 @@
 
 import requests
 
-from helpers.network_helpers import get_network_host
+from integration_tests.helpers.network_helpers import get_network_host
```

Comment:
This change is making the tests fail AFAIK (happened to me too)

Reply:
Thanks @AntoineRR 😄 I was losing my hair over this 🤣